components/sysutils/virtualbox/files/Makefile

```diff
@@ -1,34 +1,161 @@
-BINDIR = out/solaris.amd64/release/bin
-DRVS = vboxnet vboxdrv vboxflt vboxusb vboxusbmon
+PREFIX = /opt/VirtualBox
```

Comment:
is this file used?

Reply:
I do not think so but I left it since you added it, just in case.
Should be cleaned up just before merge.